sdk/python/feast/client.py

```diff
@@ -540,8 +526,26 @@ def get_batch_features(
         Examples:
             >>> from feast import Client
             >>> from datetime import datetime
-            >>>
+            >>> # Initialize the client with desired properties
+            >>> # Client with insecure core
             >>> feast_client = Client(core_url="localhost:6565", serving_url="localhost:6566")
+            >>> # Client with secure core (Auth provider: Google OpenID)
+            >>> feast_client = Client(
+            >>>     core_url="localhost:6565",
+            >>>     serving_url="localhost:6566",
+            >>>     core_enable_auth=True,
+            >>>     core_auth_provider="google")
+            >>> # Client with secure core (Auth provider: OAuth)
+            >>> feast_client = Client(
```

Comment:
I am not sure how I feel about dumping all this security information here in `get_batch_features`. I think it can be left for a separate section in our documentation or it can be documented as part of Client initialization.

Reply:
Will remove this, Client initialization has the documentation already.